优化README.md (#183)
LNSSPsd/PhoenixBuilder: README.md

Language tab

[English](./README.md) | [中文简体](./README.zh-CN.md)

# PhoenixBuilder

[![license](https://img.shields.io/badge/License-AGPL%203.0-brightgreen.svg?style=flat-square)](https://github.com/LNSSPsd/PhoenixBuilder/blob/main/LICENSE)

PhoenixBuilder is a multifunctional structure generating tool for Minecraft China Edition based on gophertunnel.

## Table of Contents

- [Install](#Install)
- [Usage](#Usage)
- [Documents](#Documents)

## Install

Prebuilt binaries available in the [Releases tab](https://github.com/LNSSPsd/PhoenixBuilder/releases/)

> If your device is Linux-based, you should use Ubuntu 22.04.Because that those prebuilt binaries require a relatively high `Libc6` version.

Reviewer: ```suggestion
If your environment is Linux-based, you may want to use a newer GLibc since our prebuilt binaries for Linux was built on Ubuntu 22.
Or, you can clone this repository and build on your device.
```

# PhoenixBuilder

[![license](https://img.shields.io/badge/License-AGPL%203.0-brightgreen.svg?style=flat-square)](https://github.com/LNSSPsd/PhoenixBuilder/blob/main/LICENSE)

> PhoenixBuilder is a multifunctional structure generating tool based on gophertunnel which is for the Bedrock Edition of Minecraft (which forked by NetEase).

Reviewer 2: The main target is weakened, and decorative part of sentence became the object.
This sentence is describing PhoenixBuilder, but turned to describe gophertunnel, and finally MCBE.

> [Here's the Offical Website](https://fastbuilder.pro)

Reviewer 2: ```suggestion
```
It's better to avoid describing non-free(/open source) stuff.

## Table of Contents

- [Install](#Install)
- [Usage](#Usage)
- [Documents](#Documents)

## Install

> You can download the newest Release for your device [here](https://github.com/LNSSPsd/PhoenixBuilder/releases) to use it.

Reviewer 2: ```suggestion
Prebuilt binaries available in the Releases tab
```

> If your device is Linux-based, you should use Ubuntu 22.04 or you'll meet runtime error.

Reviewer 2: ```suggestion
Note: Those prebuilt binaries require a relatively high `libc6` version.
```

> Linux platforms Install/Update sh command(This command also applicable for other *nix platforms (iOS, macOS, Android, etc.).):

Reviewer 2: ```suggestion
```

```sh
bash -c "$(curl -fsSL https://storage.fastbuilder.pro/install.sh)"
```

> 

Reviewer 2: Better to avoid using install script as it may pollute/mess up the system environment.
